src/main/java/org/umssdiplo/automationv01/core/managepage/SHAccidentFormModal.java

```diff
@@ -0,0 +1,84 @@
+package org.umssdiplo.automationv01.core.managepage;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.umssdiplo.automationv01.core.utils.CommonEvents;
+
+public class SHAccidentFormModal extends BasePage {
+
+    @FindBy(id = "mat-dialog-1")
```

Comment:
avoid to map all the web element, just for your pull request, which of them you will use?